### DISCUSSION TITLE
Is there a way to --legacy-peer-deps?

### DISCUSSION THREAD

**Discussion by richardpickett** [raineorshine/npm-check-updates#1381] [Q&A]
my .ncurc:

`{
  "interactive": true,
  "install": "always",
  "format": [
    "group",
    "time"
  ]
}`

I like it that it always installs after interactive, and this works on 90% of my projects.

But I have a couple projects that the only way `npm i` works is if I use `--legacy-peer-deps`, so after `ncu` I always have to run it by hand.

Is there a way to save that in .ncurc?

**Comment by raineorshine** [comment]
The way to do this would be for us to add a new `--installCommand` option that allows users to customize the install command. I don't have availability to do this work at the moment, but I'm open to PR's.

In the mean time, I might suggest using a custom npm script that runs `ncu -u && npm install --legacy-peer-deps`. Still, I recognize that it is not ideal to have to use a different command on some projects rather than others.

**Reply by davitacols** [reply]
install enhanced peer deps globally by using..
# Install globally
npm install -g epd

# Or with yarn
yarn global add epd

# Or with pnpm
pnpm add -g epd

after that use 
epd install ...
epd run dev
epd run start

**Reply by davitacols** [reply]
https://github.com/davitacols/epd
